src/main/java/org/gatorgradle/display/CommandOutputSummary.java

```diff
@@ -186,16 +186,22 @@ public void uploadOutputSummary(List<CheckResult> failed, List<CheckResult> all)
 
     HttpURLConnection con = null;
     try {
-      String endpoint = GatorGradleConfig.get().getReportEndpoint();
+      // get report endpoint and api key from environment variable
+      String endpoint = System.getenv("ENDPOINT");
```

Comment:
I think the endpoint should probably be configuration in the config, perhaps in addition to an environment variable (the env var could override the other)

Reply:
These environment variables should also have `GATOR` prepended to them, or something similar, so that it is clear what they are for, and they don't collide with other environment variables for other services.

Reply:
There's a caveat to the above: if for some reason we need to keep the endpoint secret, it might be a good idea to require an environment variable. If not, it would probably make more trouble for the user if the endpoint needed to be in an environment variable.